bootloader/grub.go

```diff
@@ -30,12 +30,19 @@ import (
 
 type grub struct {
 	rootdir string
+
+	basedir string
 }
 
 // newGrub create a new Grub bootloader object
 func newGrub(rootdir string) Bootloader {
 	g := &grub{rootdir: rootdir}
-	if !osutil.FileExists(g.ConfigFile()) {
+	switch {
+	case osutil.FileExists(filepath.Join(g.rootdir, "/boot/grub/grub.cfg")):
+		g.basedir = "/boot/grub"
+	case osutil.FileExists(filepath.Join(g.rootdir, "/EFI/ubuntu/grub.cfg")):
```

Comment:
Test needs to check if we install the right grub.cfg to the boot/recovery partition.